pkl-core/src/main/java/org/pkl/core/stdlib/test/report/JUnitReport.java

```diff
@@ -191,7 +197,11 @@ private VmTyped makeCdata(String text) {
     return new VmTyped(VmUtils.createEmptyMaterializedFrame(), clazz.getPrototype(), clazz, attrs);
   }
 
-  public static String renderXML(String indent, String version, VmDynamic value) {
+  private String stripColors(String str) {
+    return str.replaceAll("\u001B\\[[;\\d]*m", "");
```

Comment:
Let's use _either_ `\033` _or_ `\u001b`
```suggestion
    return str.replaceAll("\u033\\[[;\\d]*m", "");
```